### PR TITLE
Update to OWNERS_ALIASES

### PR DESCRIPTION
**Pull request by mazzystr**
* Update needed so I can use CNCF tools
* Pep is no longer assigned to project KubeVirt so I'm removing him as well

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -10,7 +10,7 @@ aliases:
       - stu-gott
       - SchSeba
       - fabiand
-      - codificat
```
Comment (rmohr) [inline OWNERS_ALIASES:13]: @mazzystr do you want to move pep to the emeritus section?

Comment (mazzystr) [reply OWNERS_ALIASES:13]: yes please

**Pull request merged**